lib/Zonemaster/CLI.pm

```diff
@@ -623,6 +623,10 @@ sub run {
         if ( $self->test and @{ $self->test } > 0 ) {
             foreach my $t ( @{ $self->test } ) {
                 my ( $module, $method ) = split( '/', lc($t), 2 );
+                if ( $module =~ /^([a-z]+)[0-9]+$/ ) {
```

Comment:
Shouldn't this regex use the `i` flag?

Reply:
Note the `lc` in `lc($t)`.

Reply:
But to prevent "basic01/garbage" to be taken as "basic/basic01", I suggest
```
                my ( $module, $method ) = split( '/', lc($t), 2 );
                if ( $module =~ /^([a-z]+)[0-9]+$/  and !$method) {
```

Reply:
We should also forbid "basic01/" and "basic01/0".